hermes-mqtt/src/lib.rs

```diff
@@ -560,6 +560,9 @@ impl AudioServerFacade for MqttToggleableComponentFacade {
         { bytes.wav_bytes });
     s!(subscribe_play_finished<PlayFinishedMessage>(site_id: String) { &HermesTopic::AudioServer(Some(site_id), AudioServerCommand::PlayFinished) });
     s!(subscribe_all_play_finished<PlayFinishedMessage> &HermesTopic::AudioServer(Some("+".into()), AudioServerCommand::PlayFinished););
+    p_bin!(publish_play_bytes_streaming(play_bytes_streaming_message: PlayBytesStreamingMessage)
+       { &HermesTopic::AudioServer(Some(play_bytes_streaming_message.site_id.clone()), AudioServerCommand::PlayBytesStreaming(play_bytes_streaming_message.play_bytes_id.clone(), play_bytes_streaming_message.chunk_nbr.to_string(), if play_bytes_streaming_message.is_last_chunk { "t".to_string() } else { "f".to_string() })) }
```

Comment:
not sure if I like the `t` and `f` use `0` and `1` if you whant to stick to single chars, or a full `true` and `false`